packages/vue/src/components/atoms/SfLoader/SfLoader.vue

```diff
@@ -1,9 +1,15 @@
-<template>
-  <div class="sf-loader">
+<template functional>
+  <div
+    :ref="data.ref"
+    :class="[data.class, data.staticClass, 'sf-loader']"
+    :style="[data.style, data.staticStyle]"
+    v-bind="data.attrs"
+    v-on="listeners"
+  >
     <transition name="sf-fade" mode="out-in">
       <!--@slot Slot for the actual content being loaded -->
-      <slot v-if="!loading"></slot>
-      <div v-else class="sf-loader__overlay">
+      <slot v-if="!props.loading"></slot>
+      <div v-else :class="[data.class, data.staticClass, 'sf-loader__overlay']">
```

Comment:
Same problem with class inheritance

Reply:
Also changed.

Reply:
I removed nested classes like this also from other components. Is that alright?

Reply:
yes @AdamPawlinski :)